### PR TITLE
Fix wrapobject appearance

### PR DESCRIPTION
**Pull request by aymanhab**
Fixes issue https://github.com/opensim-org/opensim-gui/issues/482 reported by beta tester

### Brief summary of changes
updateFromXML has been modified to always create an Appearance node for old WrapObjects because WrapObject Appearance defaults  (cyan, half transparent) are different from a default Appearance object (white, opaque).

### Testing I've completed
Tests pass and GUI built with this change do in fact keep Appearance correctly post (de)serialization.

### Looking for feedback on...
modifying updateFromXML 

### CHANGELOG.md (choose one)

- no need to update because bug fix

The Doxygen for this PR can be viewed at http://myosin.sourceforge.net/?C=N;O=D; click the folder whose name is this PR's number.

### CODE REVIEW

**Comment by chrisdembia** [comment]
FYI I updated the PR description to include a link to https://github.com/opensim-org/opensim-gui/issues/482

**Comment by aymanhab** [comment]
Test failures unrelated. Ready for review.

**Comment by aymanhab** [comment]
Basically the change in this PR is to modify the old updateFromXML code so that an Appearance node is always added to WrapObject. This is necessary since the default Appearance object has different defaults (White, opaque, visible) from those assumed by WrapObject (Cyan, half-transparent-visible).

File: OpenSim/Simulation/Wrap/WrapObject.cpp [hunk 1]
```diff
@@ -241,29 +237,25 @@ void WrapObject::updateFromXMLNode(SimTK::Xml::Element& node,
...
                 }
                 node.removeNode(visObjIter);
             }
             if (visibleNode.getValue() != "") {
                 appearanceNode.insertNodeAfter(appearanceNode.element_end(),
                         visibleNode);
-                appearanceModified = true;
             }
-
+            // Regardless, we'll add Appearance since default for WrapObject is different from
```
Comment (aseth1) [inline OpenSim/Simulation/Wrap/WrapObject.cpp]: Regardless of what?
Minor, comment exceeds 80cols.

**Comment by aymanhab** [comment]
Thanks for the quick review @aseth1 I updated the comment to clarify.

**Comment by aymanhab** [comment]
I'll close this since I reproduced the issue with a 4.0 wrap object, so it's not model conversion issue. Thanks for the review anyway.

**Pull request closed without merging**